index.bs

```diff
@@ -1607,58 +1652,61 @@ ceremony, a [RP] MUST proceed as follows:
 1. Perform JSON deserialization on the {{ScopedCredentialInfo/clientDataJSON}} field of the {{ScopedCredentialInfo}} object to
     extract the [=client data=] |C| claimed to have been used for the credential's attestation.
 
-2. Verify that the {{ClientData/challenge}} in |C| matches the challenge that was sent to the authenticator in the
+1. Verify that the {{ClientData/challenge}} in |C| matches the challenge that was sent to the authenticator in the
     {{makeCredential()}} call.
 
-3. Verify that the {{ClientData/origin}} in |C| matches the [RP]'s origin.
+1. Verify that the {{ClientData/origin}} in |C| matches the [RP]'s origin.
 
 4. Verify that the {{ClientData/tokenBinding}} in |C| matches the [=Token Binding ID=] for the TLS connection over
     which the attestation was obtained.
 
-5. Verify that the {{ClientData/extensions}} in |C| is a proper subset of the extensions requested by the RP.
+1. Verify that the {{ClientData/extensions}} in |C| is a proper subset of the extensions requested by the RP.
 
-6. Compute the hash of {{ScopedCredentialInfo/clientDataJSON}} using the algorithm identified by
+1. Compute the hash of {{ScopedCredentialInfo/clientDataJSON}} using the algorithm identified by
     <code>|C|.{{ClientData/hashAlg}}</code>.
 
-7. Perform CBOR decoding on the {{ScopedCredentialInfo/attestationObject}} field of the {{ScopedCredentialInfo}} structure to
-    obtain the attestation statement format |fmt|, the [=authenticator data=] |authData|, and the attestation statement
-    |attStmt|.
+1. Perform CBOR decoding on the {{ScopedCredentialInfo/attestationObject}} field of the {{ScopedCredentialInfo}} structure to
+    obtain the Credential ID, the attestation statement format |fmt|, the [=authenticator data=] |authData|, and the 
+    attestation statement |attStmt|.
+
+1. Store the Credential ID if {{ScopedCredentialOptions/keyStorage/onRelyingParty}} value is supplied in the 
+    <code>{{options}}.{{ScopedCredentialOptions/keyStorage}}</code> parameter. 
```

Comment:
Please delete this. It is not well-defined. It is not clear what "store" means here, and it is not necessary.

The credential ID must *always* be stored on a successful registration, as specified already in step 13. It does not matter if the credential is resident or not.